dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -884,3 +884,28 @@ CWWKD1086.named.params.unused.explanation=The Param annotation can only be used
  when the query has named parameters.
 CWWKD1086.named.params.unused.useraction=Remove the Param annotation or switch \
  the query to use named parameters.
+
+CWWKD1087.null.param=CWWKD1087E: The {0} parameter of the {1} method of the \
+ {2} repository interface cannot be a null value.
+CWWKD1087.null.param.explanation=The value must not be null.
+CWWKD1087.null.param.useraction=Supply a non-null value.
+
+CWWKD1088.empty.sorts=CWWKD1088E: The sort criteria that is supplied via the \
```

Comment:
The sort criteria that are specified by the {0} parameter of the {1} method of the {2} repository interface are empty.

The sort criteria must not be empty.

Specify a value for the sort criteria.

Reply:
updating as requested